client/src/containers/sidebar/index.tsx

```diff
@@ -0,0 +1,90 @@
+"use client";
+import { FC } from "react";
+
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+import { cn } from "@/lib/utils";
+
+import Header from "@/containers/header";
+import SidebarAccordion from "@/containers/sidebar/sidebar-accordion";
+
+import {
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import { Button } from "@/components/ui/button";
+
+const Sidebar: FC = () => {
+  const currentPage = usePathname().startsWith("/sandbox")
+    ? "sandbox"
+    : "explore";
+  let AccordionComponent = null;
+
+  if (currentPage === "explore") {
+    AccordionComponent = (
+      <SidebarAccordion
+        key="sidebar-accordion-explore"
+        defaultValue={["explore-sections"]}
+      >
+        <AccordionItem value="explore-filters">
+          <AccordionTrigger>Filters</AccordionTrigger>
+          <AccordionContent>filters here...</AccordionContent>
+        </AccordionItem>
+        <AccordionItem value="explore-sections">
+          <AccordionTrigger>Sections</AccordionTrigger>
+          <AccordionContent></AccordionContent>
+        </AccordionItem>
+      </SidebarAccordion>
+    );
+  }
+
+  if (currentPage === "sandbox") {
+    AccordionComponent = (
+      <SidebarAccordion
+        key="sidebar-accordion-sandbox"
+        defaultValue={["sandbox-settings", "sandbox-filters"]}
+      >
+        <AccordionItem value="sandbox-settings">
+          <AccordionTrigger>Settings</AccordionTrigger>
+          <AccordionContent>settings here...</AccordionContent>
+        </AccordionItem>
+        <AccordionItem value="sandbox-filters">
+          <AccordionTrigger>Filters</AccordionTrigger>
+          <AccordionContent>filters here...</AccordionContent>
+        </AccordionItem>
+      </SidebarAccordion>
+    );
+  }
```

Comment:
Based on designs, we have filters in both pages and only sections in explore and settings in sandbox. I would simplify this a little by rendering a single Accordion and then choosing what items (AccordionItem) should be rendered inside based on the page as you were already doing.

Feel free to render directly the accordion in the DOM so you can save the variable. It won't be possible to have a _null_ accordion as the Sidebar, so far, is rendered exclusively in /explore and /sandbox pages.